spec/lib/rex/proto/kerberos/model/enc_kdc_response_spec.rb

```diff
@@ -304,6 +345,18 @@
         enc_kdc_response.decode(enc_as_resp)
         expect(enc_kdc_response.sname.name_string).to eq(['krbtgt', 'DEMO.LOCAL'])
       end
+
+      it "decodes the pa_data correctly" do
+        enc_kdc_response.decode(enc_as_resp)
+        expected_pa_data = [
+          Rex::Proto::Kerberos::Model::PreAuthData.new(
+            type: Rex::Proto::Kerberos::Model::PA_SUPPORTED_ETYPES,
+            # SupportedEnctypes: 0x0000001f, des-cbc-crc, des-cbc-md5, rc4-hmac, aes128-cts-hmac-sha1-96, aes256-cts-hmac-sha1-96
+            value: "\x1F\x00\x00\x00"
+          )
```

Comment:
It looks like there's a few dedicated pa type objects in framework for request scenarios, but they aren't wired up to the decoding scenarios:

- pre_auth_enc_time_stamp.rb
- pre_auth_pac_request.rb

I've opted to leave that out for now.